app/components/page_list_component/view.html.erb

```diff
@@ -40,31 +40,28 @@
                 </dt>
 
                 <dd class="govuk-summary-list__value">
-                  <% if condition.has_errors_for_field?(:answer_value) %>
-                    <p class="govuk-!-margin-2">
-                      <%= t("page_conditions.condition_check_page_text_with_errors", check_page_text: question_text_for_page(condition.check_page_id)) %>
-                    </p>
-                    <%# List all errors, including for other fields %>
+                  <p class="govuk-!-margin-2">
+                    <%= t("page_conditions.#{condition.validation_errors.length > 1 || condition.has_errors_for_field?(:answer_value) ? "condition_check_page_text_with_errors" : "condition_check_page_text"}", check_page_text: question_text_for_page(condition.check_page_id)) %>
+                  </p>
+
+                  <% if condition.validation_errors.length > 1 %>
```

Comment:
```suggestion
                  <% if condition.validation_errors.any? %>
```

Reply:
This is for if there are multiple errors (i.e. 2 or more. Might change it to >= 2 to make that clearer, unless there's a nice Ruby method for doing this

Reply:
Made it even clearer by adding a method to the conditions model